keyboards/ah/haven60/readme.md

```diff
@@ -0,0 +1,27 @@
+# haven60
+
+![haven60](https://i.imgur.com/Uja750Oh.png)
+
+*A short description of the keyboard/project*
```

Comment:
This needs to be filled in

Reply:
updated

Reply:
Not sure why we would still want the template text...
```suggestion
```